### PR TITLE
JS2 week1 homework

### PR DESCRIPTION
**Pull request by shpomp**
voila

### CODE REVIEW

**Comment by shpomp** [comment]
Please ignore the "updated CSS". Don't know why it is here :D 
Was trying to fix a wrong HW submission via a temporary branch, somehow still unwanted things appear here.

File: javascript/javascript2/week1/in-class/inclass.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  {
+    name: "You, me, we and us",
+    //imageUrl: "https://picsum.photos/536/354",
+  },
+  {
+    name: "Breakfast news - Dinner edition",
+  },
+];
+
+const varUl2 = document.createElement("ul");
```
Comment (piyushpalod) [inline javascript/javascript2/week1/in-class/inclass.js:45]: try to use some better variable name , may be podcast list ?

File: javascript/javascript2/week1/in-class/inclass.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+varUl2.innerHTML = "<h1>The podcasts</h1>";
+pageBody.appendChild(varUl2);
+
+for (let i = 0; i < podcasts.length; i++) {
+  let podcastInList = document.createElement("li");
+  let h2InPodcastList = document.createElement("h2");
```
Comment (piyushpalod) [inline javascript/javascript2/week1/in-class/inclass.js:51]: Improve variable name , what is 'h2In' :)

File: javascript/javascript2/week1/spiritanimal.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+let resetButton = document.createElement("button");
+resetButton.setAttribute("id", "resetButton");
+resetButton.innerText = "reset";
+resetButton.addEventListener("click", resetButtonClicked);
+const generatedNameBox = document.createElement("h2");
+
+function userNameAdded() {
+  userNameVar = document.getElementById("userName").value;
+
+  if (userNameVar.length === 0) {
```
Comment (piyushpalod) [inline javascript/javascript2/week1/spiritanimal.js:96]: Nice check for null ! good work.

File: javascript/javascript2/week1/spiritanimal.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+function newNameButtonClicked() {
+  generatedSpiritAnimalName = ` the ${
+    spiritAdjectivesArr[Math.floor(Math.random() * spiritAdjectivesArr.length)]
+  } ${spiritAnimalsArr[Math.floor(Math.random() * spiritAnimalsArr.length)]}`;
+  generatedNameBox.innerText = userNameVar + generatedSpiritAnimalName;
+  newNameButton.innerText = "try again?";
+}
+
+function resetButtonClicked() {
+  location.reload();
```
Comment (piyushpalod) [inline javascript/javascript2/week1/spiritanimal.js:124]: very good , Nice use of reload.

**Review by piyushpalod — APPROVED** [review]
Great work, I saw working solution and its looking nice.

File: javascript/javascript2/week1/in-class/inclass.js (hunk 1 above)
Comment (shpomp) [reply javascript/javascript2/week1/in-class/inclass.js:45]: yes, just panicked while doing DOM for the first time :D

**Pull request merged**